fw/tempesta_fw.h

```diff
@@ -38,6 +38,10 @@
 
 #define DEF_MAX_PORTS		8
 
+enum {
+	SS_F_HTTT2_FRAME_TRAILER_HEADERS = 0x40,
```

Comment:
All skb flags are located in skbuff.h

Reply:
yes, but that involves the linux-5.10.35-tfw repo, so I place it here temporarily as a placeholder.

Reply:
You should make appropriate changes in  linux-5.10.35-tfw and change linux-5.10.35.patch according them

Reply:
yes, I know.